src/main/java/core/basesyntax/DateConvertor.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class DateConvertor {
+    public LocalDate convert(String strDate) {
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate beginningOfThePeriod = LocalDate.parse(strDate, dateTimeFormatter);
```

Comment:
maybe not best name, because we don't actually know what `strDate` will come to the method